users/models.py

```diff
@@ -1,8 +1,83 @@
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
+from django.contrib.auth.models import PermissionsMixin
+from django.db import models
 from django.utils.translation import ugettext_lazy as _
-from helusers.models import AbstractUser
+from uuid import uuid4
 
 
-class User(AbstractUser):
+class UserManager(BaseUserManager):
+    use_in_migrations = True
+
+    def create_user(self, email, password=None, **extra_fields):
+        """
+        Creates and saves a user with the given email and password.
+        """
+        if not email:
+            raise ValueError("The given email must be set")
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        if password is None:
+            user.set_unusable_password()
+        else:
+            user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault("is_superuser", True)
+        extra_fields.setdefault("is_staff", True)
+
+        if extra_fields.get("is_superuser") is not True:
+            raise ValueError("Superuser must have is_superuser=True.")
+
+        return self.create_user(email, password, **extra_fields)
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    CONTACT_LANGUAGE_CHOICES = [
+        ("fi", "Finnish"),
+        ("sv", "Swedish"),
+        ("en", "English"),
+    ]
+
+    uuid = models.UUIDField(primary_key=True, default=uuid4, editable=False)
```

Comment:
Another way is to do a Profile model and then link it to the user model: https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/models.py#L79. I don't have a strong opinion on how to do it, but it could be good in the future, that we don't make bigger changes to Django's user model as Django can make changes to it.

Reply:
I think this is a good approach in genera (composition over inheritance). I've done so in the latest commit. Although apparently the current `User` model in `develop` seems to be slightly modified already.